ru/mail/chrome/messenger/messengercompose/mailComposeEditorOverlay.dtd

```diff
@@ -1,4 +1,4 @@
-<!ENTITY attachImageSource.label         "Вложить это изображение в сообщение">
+﻿<!ENTITY attachImageSource.label         "Вложить это изображение в сообщение">
```

Comment:
This file was in the ANSI encoding and I have changed it to the UTF-8 one. 
Otherwise you will see the wrong characters.